src/charmed_kubeflow_chisme/testing/charm_security_context.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Utilities for testing security context and user privileges in charms."""
+import subprocess
+
+import lightkube
+from lightkube.resources.core_v1 import Pod
+
+
+def generate_container_uid_map(metadata_yaml: dict, juju_user_id: int = 170):
```

Comment:
suggestion (non-blocking): perhaps we can rename the utility to something less ambiguous like `generate_container_securitycontext_map`?